credits.txt

```diff
@@ -0,0 +1,4 @@
+@TODO - Add these in about section -
```

Comment:
We need an ABOUT file for this.

Reply:
hey, @pombredanne 
can you share an example or basic Outline for this ABOUT file
@AyanSinhaMahapatra , maybe you've some inputs on this

Reply:
@OmkarPh sorry I missed this, 
- for a package this might look like: https://github.com/nexB/thirdparty-packages/blob/main/pypi/Babel-2.10.1.tar.gz.ABOUT
- for a file: it could be like: https://github.com/nexB/scancode-toolkit/blob/develop/src/scancode/pool.py.ABOUT

Basically a download_url with license/copyright information and any other info you can find for giving credits/attribution. You might also need to add a copy of the license file if applicable, like the second example above in some cases depending on the license.

Reply:
I've created about file for the image asset https://github.com/OmkarPh/scancode-workbench/blob/v4.0-react-typescript/assets.ABOUT
I also added it in attribution window

Rest, I couldn't find any piece of code that I've directly copied from the internet
except for the official docs

Reply:
Use `license` instead of `usage_terms`, and the about file should be just beside the file which you are describing, and the filename would be: `addFiles.png.ABOUT`.

The terms and conditions in https://storyset.com/terms looks good to me, since it says `The authorization to use the Storyset Content shall be free provided that any use of the Storyset Content by the User is credited to the Company/Website as stated by the Company from time to time.` but  this needs to also be reviewed by @pombredanne .

Thanks!

Reply:
The TOS are significantly more complex than the "Authorization to use...." sentence so we need to look at the whole TOS. I do not see a problem, but we need to review in more detail.

Reply:
updated: https://github.com/OmkarPh/scancode-workbench/blob/v4.0-react-typescript/src/assets/images/Addfiles.png.ABOUT